tests/function/test_config.py

```diff
@@ -490,3 +490,108 @@ def test_http_proxy_in_virtwho_conf(self, virtwho, globalconf, proxy_data):
                     and result['thread'] == 1)
 
             globalconf.delete('system_environment')
+
+
+@pytest.mark.usefixtures('globalconf_clean')
+@pytest.mark.usefixtures('hypervisor_create')
+class TestSysConfiguration:
+    @pytest.mark.tier1
```

Comment:
Updated, there already have the RHEL8 and RHEL9 marks

Reply:
I think it's better to use the `notRHEL9` here, because in the pipeline jobs we define the marks with `and not [mark]`, such as when we run in rhel8, we will filter as `if ('RHEL8' in compose){marks += "and not notRHEL8 "}`.

And surely we can use `if ('RHEL8' in compose){marks += "and not rhel9 "}`, but if rhel10 come, and we may need to maintain the three lines together for some time, at that time we may need to change some cases to `if ('RHEL8' in compose){marks += "and not rhel9 and not rhel10 "}`.

What do you think.

Reply:
And then, we need to add the notRHEL9 notRHEL10 for these cases, maybe not a perfect way :)